servicetalk-dependencies/build.gradle

```diff
@@ -39,6 +40,7 @@ dependencies {
     // Ideally these come from the Netty BOM
     api "io.netty.incubator:netty-incubator-transport-native-io_uring:$nettyIoUringVersion"
 
+    api "io.opentelemetry:opentelemetry-api:$opentelemetryVersion"
```

Comment:
This section is only for modules that do not have their own boms. Since `opentelemetry-api` is listed inside `opentelemetry-bom`, we don't need this line.